<?xml version="1.0"?>
<launch>
	<node pkg="bluerov2_dobmpc" type="bluerov2_ctrl_node" name="bluerov2_ctrl_node" output="screen">
        <rosparam command="load" file="$(find bluerov2_dobmpc)/config/ctrl.yaml" />
	</node>

</launch>